tests/radiation_transport/test_advective_transport.py

```diff
@@ -134,6 +164,37 @@ def test_plotting(self):
         ax = self.solver.plot(show=True)
         assert len(ax.lines) > 2
 
+    def test_analyse_SN(self, caplog):
+        fw = deepcopy(self.solver.first_wall)
+        self.solver.flux_surfaces_ob_hfs = []
```

Comment:
If you look at the naming in the double null test, it has changed slightly as per #870 so I think it will be `ob_up` and `ob_down`